docs/docs/artifact-store.md

```diff
@@ -4,39 +4,38 @@ title: Artifact Store
 
 # Artifact Store
 
-Deployments inside a distributed system need a location, where application specific resources can be found.
-In Marathon we call this place the artifact store.
+Deployments inside a distributed system need a location where application-specific resources can be found. In Marathon, we call this place the artifact store.
 
 
 ## Usage
 
-The use of the Marathon Artifact Store functionality is completely optional.
-All functionality and goals can be accomplished manually as well.
-Marathon tries to simplify several use cases.
+Use of the Marathon artifact store is completely optional: you can create all of the artifact atore functionality manually as well. The Marathon artifact store simplifies several use cases.
 
 
 ## Artifact Store Backend
 
-Marathon supports different storage system, that can be used as artifact store.
-The type of the artifact store is configured via the command line.
-Example: start with --artifact_store hdfs://localhost:54310/path/to/store to use Hadoop DFS 
-as artifact storage backend.
+Marathon supports several different storage system backends for the artifact store.
 
+You can specify the storage system backend on the command line. To use Hadoop DFS, for example, use the following command line flag:
 
-## Artifact REST endpoint
+ --artifact_store hdfs://localhost:54310/path/to/store 
 
-There is a special endpoint to access and manipulate the artifacts in the artifact store.
-The URL's of the created artifacts can be used as URI's of an application definition.
 
-### Upload an artifact to the artifact store
+## Artifact REST Endpoint
 
-Upload an artifact to the artifact store.
-A multipart form upload request has to be performed.
-The form parameter name has to be ```file```.
-The filename used in the artifact store, is the same as given by the form parameter.
-The response holds the URL of the artifact in the artifact store in the Location Header.
+A special endpoint enables you to access and manipulate the artifacts in the artifact store. The URLs of the created artifacts can be used as URIs of an application definition.
+
+See the [REST API documentation](http://mesosphere.github.io/marathon/docs/generated/api.html#v2_artifacts) for more information.
+
+### Upload an Artifact to the Artifact Store
+
+To upload an artifact to the artifact stopre, you must perform a multipart form upload request.
```

Comment:
No. That step has to be performed, before the application is created, that uses those artifacts.